std/src/lib.rs

```diff
@@ -124,6 +130,7 @@ macro_rules! cfg_into_iter {
 #[macro_export]
 macro_rules! cfg_chunks {
     ($e:expr, $size:expr) => {{
+        use $crate::rayon::prelude::*;
```

Comment:
just wanted to check, does crate resolve as this crate, or the crate of where the macro is being called?

Reply:
Within `ark-std`, this resolves as `crate::`, and outside `ark-std`, this resolves as `ark_std::`